QuickStart/inv_NACA0012.cfg

```diff
@@ -22,6 +22,8 @@ MATH_PROBLEM= DIRECT
 % Restart solution (NO, YES)
 RESTART_SOL= NO
 
+%CONFIG_LIST = (inv_NACA0012.cfg)
```

Comment:
This is just something that slipped in because I changed it for some tests. I'll remove it from the quickstart. However, once the feature is fully adapted I will incorporate the options to the template config.